python_modules/libraries/dagster-tableau/dagster_tableau/resources.py

```diff
@@ -127,6 +127,7 @@ def cancel_job(
         """Cancels a given job."""
         return self._server.jobs.cancel(job_id)
 
+    @superseded(additional_warn_text="Use `refresh_and_poll` at the resource-level instead.")
```

Comment:
"at the resource-level" is a bit confusing -- maybe this should just say something along the lines of "on the tableau resource instead."?

up to you

Reply:
Sure let's do that